__tests__/praxis/arthur-deviation-biases/unit.ts

```diff
@@ -51,24 +52,24 @@ describe('ArthurDeviationBiases Class: Unit', () => {
     it('calls this.kernel() returns kernel output', () => {
       const mockWeights = 1;
       const mockDeltas = 2;
-      const mockLayer = {
+      const mockLayer: any = {
         width: 2,
         height: 2,
         deltas: mockDeltas,
         weights: mockWeights,
       };
-      const p = new ArthurDeviationBiases(mockLayer);
+      const p: any = new ArthurDeviationBiases(mockLayer);
```

Comment:
Use type for `p`

Reply:
`Type 'Mock<{}, []>' is missing the following properties from type 'IKernelRunShortcut': kernel, exec, source, Kernel, and 67 more.`
This error is due to line 63.

Reply:
`Type 'Mock<{}, []>' is missing the following properties from type 'IKernelRunShortcut': kernel, exec, source, Kernel, and 67 more.`
This error is due to line 63.

Reply:
@mubaidr any update...

Reply:
Skip this for now.